typings/index.d.ts

```diff
@@ -1185,6 +1213,10 @@ declare module 'discord.js' {
       filter: CollectorFilter<[MessageReaction, User]>,
       options?: ReactionCollectorOptions,
     ): ReactionCollector;
+    public createMessageComponentInteractionCollector(
```

Comment:
Seems like the typings for `awaitMessageComponentInteractions` is missing?

Reply:
...must have lost them in a rebase I guess? They definitely existed, will add them back